jest_setup.ts

```diff
@@ -38,3 +38,8 @@ jest.mock('react-native/Libraries/Utilities/PixelRatio.js', () => ({
 
 // @ts-ignore
 global.__reanimatedWorkletInit = jest.fn()
+
+// avoid side effects from importing Torus dependencies while testing (doing so unmocks fetch, even for unrelated tests)
+// TODO may need to remove once we update to use latest Torus/Web3Auth API https://linear.app/valora/issue/ACT-876
+jest.mock('@toruslabs/torus.js', () => jest.fn())
+jest.mock('@toruslabs/fetch-node-details', () => jest.fn())
```

Comment:
Not a blocking comment, but I think all our global mocks are in the __mocks__ folder.

Reply:
what about the other mocks in this file? they don't seem to be in the mocks folder

Reply:
..but I moved the mocks to __mocks__ anyway, seems like it might be a bit cleaner, and maybe there's some reason that isn't possible for some modules and that's why they're here in jset_setup, and I'm just not catching on to it..